src/position.cpp

```diff
@@ -1546,10 +1559,28 @@ Value Position::see(Move m) const {
 #endif
 
       // Add the new entry to the swap list
+#ifdef ANTI
+      if (is_anti())
+          swapList[slIndex] = -swapList[slIndex - 1] + PieceValueAnti[MG][captured];
+      else
+#endif
       swapList[slIndex] = -swapList[slIndex - 1] + PieceValue[MG][captured];
 
       // Locate and remove the next least valuable attacker
       captured = min_attacker<PAWN>(byTypeBB, to, stmAttackers, occupied, attackers);
+#ifdef ANTI
```

Comment:
For ease of reading code I may move this to `min_attacker_anti` (defined immediately above or below `min_attacker`) without risking slow-down.